pyproject.toml

```diff
@@ -39,11 +39,13 @@ docs = [
     'sphinx-book-theme',
 ]
 test = [
+    'gitpython',
     'packaging',
     'pytest',
     'pytest-cov',
     'pytest-xdist',
     'ruff>=0.9',
+    'setuptools',
```

Comment:
Where is setuptools used? I didn't see it

Reply:
Yeah, it's strange but `torch.compile` fails without setuptools

https://github.com/qubvel-org/segmentation_models.pytorch/actions/runs/12753548754/job/35545453127#step:6:3442

Reply:
<img width="1188" alt="image" src="https://github.com/user-attachments/assets/96a26a0c-a3bc-45df-a7d2-cebfeb91b9c1" />

Reply:
Should we add it to lib dependencies as well?

Reply:
Good to know. I opened a PR to fix this in PyTorch: https://github.com/pytorch/pytorch/pull/144763. For now, let's just keep it for testing only. It's really a bug in PyTorch, not in SMP. We can remove it once it's fixed upstream.

Reply:
Ok, thanks!